phpstan.neon

```diff
@@ -12,5 +13,7 @@ parameters:
 
     symfony:
         container_xml_path: 'var/cache/dev/App_KernelDevDebugContainer.xml'
+        console_application_loader: src/Command/console-application.php
```

Comment:
I don't really like theses .php files in the src dir. Could you move them in the `var` dir? thanks